packages/ssi-service/package.json

```diff
@@ -40,5 +44,19 @@
     "webpack-cli": "3.3.11",
     "webpack-dev-server": "3.11.0",
     "whatwg-fetch": "3.0.0"
+  },
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/1-Platform/one-platform.git"
```

Comment:
I am wondering why url starts with "git+".

Reply:
Actually I'm not sure about that either. That was added by `npm init`.